### ISSUE TITLE
[ALM Accelerator - QUESTION] Clarification on ALM AA4PP app

### ISSUE THREAD

**Issue by NagulapallyLaxmanGoud** [microsoft/coe-starter-kit#2611]
### What is your question?

I just need a better clarification on selection of the Branches in the AA4PP app when I am trying to commit or deploying the Solution.

1. When I am trying to commit a solution, by default I am getting the content in the branch information as below 
 Should I leave it as is or if I change the branch with another existing branch under 'based on' what happens ? 
![image](https://user-images.githubusercontent.com/99319322/166007449-f7e6a118-73ce-42bc-bb03-65c0c0b06071.png)

2. When I am trying to deploy a solution, by default I am getting the content in the branch information as below 
 Should I leave it as is or if I change the target branch with another existing branch what happens ? 
If I select main as target branch I am getting an error.
If I select solutionname (Newly Created Solution) as target branch I am able to move further with next steps. Is it like we should only select the  solutionname (Newly Created Solution) as the Target Branch or can I select any of the existing Solution.

![image](https://user-images.githubusercontent.com/99319322/166009807-3f43cb80-77f0-4b1a-8148-b83824649481.png)

### What component are you experiencing the issue with?

ALM Accelerator Canvas App

### What solution version are you using?

April 2022

**Comment by mikefactorial** [comment]
Thanks for the question @NagulapallyLaxmanGoud we have made some updates in the may release to make this more obvious and to default the branches appropriately. In addition we have a task to make this clearer in the documentation. Essentially this is the process.

- First commit create new working branch from main
- Push to test is PR from working branch to solution branch (kicks off validation build)
- Complete PR kicks off test build
- Push to prod is PR from solution branch to main (kicks off production build)
- Next commit create new working branch from solution branch
- Repeat from step 2

To your specific questions 1. Yes 2. No - this is what we needed to fixed for the defaults for the May release.
